components/feature/addons/src/main/java/mozilla/components/feature/addons/AddonManager.kt

```diff
@@ -177,11 +178,13 @@ class AddonManager(
      * Enables the provided [Addon].
      *
      * @param addon the [Addon] to enable.
+     * @param source [EnableSource] to indicate why the extension is enabled.
```

Comment:
nit: default to EnableSource.USER

Why is that the case? Just curious ..

Reply:
Fixed :)
When we migrate add-ons we [disabled them with the source EnableSource.APP_SUPPORT](https://github.com/mozilla-mobile/android-components/blob/e796fa7408cd9affa6b38a3bfc07092d89bce2da/components/support/migration/src/main/java/mozilla/components/support/migration/AddonMigration.kt#L176), but users are unable to enable the newly supported add-on using source `EnableSource.USER` that's what   the engine uses by default causing this issue https://github.com/mozilla-mobile/fenix/issues/9994, for this reason, we need to enable the add-on using `EnableSource.USER` for users be able to enable/disable newly supported add-ons.